src/node_file.cc

```diff
@@ -1284,8 +1284,8 @@ int MKDirpAsync(uv_loop_t* loop,
   FSReqBase* req_wrap = FSReqBase::from_req(req);
   // on the first iteration of algorithm, stash state information.
   if (req_wrap->continuation_data == nullptr) {
-    req_wrap->continuation_data = std::unique_ptr<FSContinuationData>{
-      new FSContinuationData(req, mode, cb)};
+    req_wrap->continuation_data = std::make_unique<FSContinuationData>(
+      req, mode, cb);
```

Comment:
Side note: This continuation should have 4 spaces of indentation

Reply:
Looks like many format problem, I will run `clang-format` on this patch.